src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -500,62 +570,113 @@ describe('Schemaform save / load actions:', () => {
+          (req, res, ctx) => {
+            return res(ctx.status(200));
+          },
+        ),
+      );
       const thunk = removeInProgressForm(VA_FORM_IDS.FORM_10_10EZ, {});
       const dispatch = sinon.spy();
-      setFetchJSONFailure(global.fetch.onCall(0));
 
       return thunk(dispatch, getState).then(() => {
         expect(dispatch.calledWith(setStartOver())).to.be.true;
       });
     });
     it('attempts to remove an in-progress form', () => {
+      server.use(
+        rest.delete(
+          inProgressApi(VA_FORM_IDS.FORM_10_10EZ),
+          (req, res, ctx) => {
+            return res(ctx.status(200));
+          },
+        ),
+      );
       const thunk = removeInProgressForm(VA_FORM_IDS.FORM_10_10EZ, {});
       const dispatch = sinon.spy();
 
       thunk(dispatch, getState).then(() => {
-        expect(global.fetch.firstCall.args[0]).to.contain(
-          '/v0/in_progress_forms/1010ez',
-        );
-        expect(global.fetch.firstCall.args[1].method).to.equal('DELETE');
+        expect(expectedURL).to.contain(inProgressApi(VA_FORM_IDS.FORM_10_10EZ));
+        expect(expectedMethod).to.equal('DELETE');
       });
     });
     it('removes a form and fetches prefill data', () => {
+      const mockedData = {
+        data: {
+          id: '1',
+          type: '10-10EZ Save in Progress Delete',
+          attributes: {
+            formId: VA_FORM_IDS.FORM_10_10EZ,
+            createdAt: 'today',
+            updatedAt: 'yesterday',
+            metadata: {},
+          },
+        },
+      };
+      server.use(
+        rest.delete(
+          inProgressApi(VA_FORM_IDS.FORM_10_10EZ),
+          (req, res, ctx) => {
+            return res(ctx.status(200), ctx.json(mockedData));
+          },
+        ),
+      );
       const thunk = removeInProgressForm(VA_FORM_IDS.FORM_10_10EZ, {});
       const dispatch = sinon.spy();
-      setFetchJSONResponse(global.fetch.onCall(0));
 
       return thunk(dispatch, getState).then(() => {
-        expect(global.fetch.firstCall.args[1].method).to.equal('DELETE');
+        expect(expectedMethod).to.equal('DELETE');
         expect(dispatch.lastCall.args[0]).to.be.a('function');
       });
     });
     it('handles remove error and fetches prefill data', () => {
+      server.use(
+        rest.delete(inProgressApi(VA_FORM_IDS.FORM_10_10EZ), (req, res) => {
+          return res.networkError('Network Error');
+        }),
+      );
       const thunk = removeInProgressForm(VA_FORM_IDS.FORM_10_10EZ, {});
       const dispatch = sinon.spy();
-      setFetchJSONFailure(global.fetch.onCall(0));
 
       return thunk(dispatch, getState).then(() => {
-        expect(global.fetch.firstCall.args[1].method).to.equal('DELETE');
+        expect(expectedMethod).to.equal('DELETE');
         expect(dispatch.lastCall.args[0]).to.be.a('function');
       });
     });
     it('sets no-auth status if session expires', () => {
+      server.use(
```

Comment:
It looks like there's a lot of refactoring around switching the mocking. Is that necessary to support the change to `apiRequest()`?